Wikipedia/Code/ExploreViewController.swift

```diff
@@ -640,6 +640,8 @@ class ExploreViewController: ColumnarCollectionViewController, ExploreCardViewCo
     }
     
     func collectionViewUpdater<T: NSFetchRequestResult>(_ updater: CollectionViewUpdater<T>, didUpdate collectionView: UICollectionView) {
+		WidgetController.shared.reloadAllWidgets()
```

Comment:
I wonder if we ought to put these `reloadAllWidgets` calls somewhere closer to the Explore feed data refresh layer, maybe in `WMFExploreFeedContentController`. This one in particular I'm wondering how it will act if the Explore feed is not on screen (which you can see via Settings > Explore feed > turn off Explore tab toggle at the bottom). It may be no difference though, I would also expect `WMFExploreFeedContentController` to not be around or active if the user has disabled Explore on screen.

Reply:
I moved the placement of these reload calls into `WMFExploreFeedContentController.m`, but I did confirm those calls aren't hit (nor is this one) if the Explore tab is hidden via app settings. So a network call fallback might be in order at some point to handle that edge case.

Reply:
Update: `WMFExploreFeedContentController.m` is hit if I lower the refresh threshold. So I think we're good there, though the widget still had trouble refreshing when the Explore feed was hidden so not sure where it's getting dropped. We can look into it in a followup PR.